asistente_ladm_col/logic/quality_rules/qr_boundary_point_duplicate_records.py

```diff
@@ -0,0 +1,97 @@
+"""
+/***************************************************************************
+                              Asistente LADM_COL
+                             --------------------
+        begin                : 2022-06-12
+        git sha              : :%H$
+        copyright            : (C) 2022 by Sergio Ramírez (BSF Swissphoto)
+        email                : seralra96@gmail.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames, LayerConfig
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4011,
+                                                           QRE_IGACR4011E01)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRDuplicateBoundaryPointRecords(AbstractLogicQualityRule):
+    """
+    Check that boundary point table don't have duplicate records
+    """
+    _ERROR_01 = QRE_IGACR4011E01  # Boundary point with duplicate records
+
+    def __init__(self):
+        AbstractLogicQualityRule.__init__(self)
+
+        self._id = QR_IGACR4011
+        self._name = "Revisar que la tabla punto lindero no tenga registros repetidos"
```

Comment:
I propose to change the name of the quality rule to: "Punto Lindero no debe tener registros repetidos"

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/549/commits/974e8b3bff931f065de034f08c18c0a0244de324